Common++/header/LRUList.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <list>
+#include <cstddef>
```

Comment:
Why adding this include?

Reply:
See comment about CI fails. This is the header in which std::size_t is guaranteed to be defined by the standard.